linter.py

```diff
@@ -86,12 +86,23 @@ def lint_property_names(spec, resolver):
 
     http://zalando.github.io/restful-api-guidelines/json-guidelines/JsonGuidelines.html#must-property-names-must-be-snakecase-and-never-camelcase
     """
-    # TODO: this only iterates over global definitions (and only one level)
+    definitions = []
     for def_name, definition in spec.get('definitions', {}).items():
         if definition.get('type') == 'object':
```

Comment:
this still only works for top-level definitions (not for `array` type on top-level)